specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -1886,7 +1886,13 @@
             }
           }
         },
-        "x-ms-long-running-operation": true
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update a VM by detaching data disk": {
+            "$ref": "./examples/UpdateVMDetachDataDiskWithToBeDetachedFlag.json"
```

Comment:
Can we rename this file to UpdateVMDetachDataDiskWithToBeDetachedProperty.json

Reply:
done